tests/network/localnet/test_ovs_bridge.py

```diff
@@ -12,3 +18,29 @@ def test_connectivity_over_migration_between_ovs_bridge_localnet_vms(
 ):
     migrate_vm_and_verify(vm=localnet_ovs_bridge_client.vm)
     assert is_tcp_connection(server=localnet_ovs_bridge_server, client=localnet_ovs_bridge_client)
+
+
+@pytest.mark.ipv4
+@pytest.mark.usefixtures("nncp_localnet_on_secondary_node_nic")
+@pytest.mark.polarion("CNV-12006")
+def test_connectivity_after_interface_state_change_in_ovs_bridge_localnet_vms(
+    ovs_bridge_localnet_running_vms_one_with_interface_down,
+):
+    ovs_bridge_localnet_running_vms_one_with_interface_down[0].set_interface_state(
```

Comment:
I would prefer to know which VM is the one with the link-down link.
Can you extract above in the correct order and name it to avoid jumping to the fixture to find out?
e.g. `(vm1_with_initial_link_down, vm2) = ovs_bridge_localnet_running_vms_one_with_interface_down`.